cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -522,18 +520,19 @@ private String getClassPath(JBallerinaBackend jBallerinaBackend, Package current
 
         for (ModuleId moduleId : currentPackage.moduleIds()) {
             Module module = currentPackage.module(moduleId);
-
-            // Skip adding the path if the module doesnt contain a generated module jar
-            if (!module.documentIds().isEmpty()) {
+            try {
                 PlatformLibrary generatedJarLibrary = jBallerinaBackend.codeGeneratedLibrary(
                         currentPackage.packageId(), module.moduleName());
                 exclusionPathList.add(generatedJarLibrary.path());
+            } catch (IllegalStateException e) {
+                continue;
             }
-            // Skip adding the path if the module doesnt contain a generated testable jar
-            if (!module.testDocumentIds().isEmpty()) {
```

Comment:
I think we can have this check as is it because a testable jar will be created only if there are test documents.

Reply:
This check wasnt working when dealing with multiple modules. I removed it and used the try/catch since it would handle the exception if the jar file didnt exist.

Reply:
Shall we check why the testable jar was generated even when there were are no test files found rather than catching the exception and continuing?

Reply:
Changed the implementation back to checking the testDocuementIds instead of catching the exception.